lib/src/hooks/useAnimatedPath.ts

```diff
@@ -54,9 +54,32 @@ export const useAnimatedPath = (
   }, [path, t]);
 
   const currentPath = useDerivedValue<SkPath>(() => {
-    if (t.value !== 1 && path.isInterpolatable(prevPath)) {
-      return path.interpolate(prevPath, t.value) || path;
+    if (t.value !== 1) {
+      if (!path.isInterpolatable(prevPath)) {
+        // Match floating-point numbers in a string and normalize their precision as this is essential for Skia to interpolate paths
+        // Without normalization, Skia won't be able to interpolate paths in Pie slice shapes
+        const normalizePrecision = (path: string): string =>
```

Comment:
is there value in just always normalizing the paths?

Reply:
Good question! I don't think it would be harmful, but I sort of want to keep this special path separate for some reason. I think maybe if we come across another instance where we need to massage the data to be interpolatable we move to something like [this flubber library RN Skia recommends](https://shopify.github.io/react-native-skia/docs/animations/hooks/#usepathinterpolation)